docs/user/build-notifications.rst

```diff
@@ -1,12 +1,12 @@
-Build Notifications and Webhooks
-================================
+Build notifications via webhooks and email
+==========================================
```

Comment:
@ericholscher 

re: naming is hard

I changed the feature page to have this title instead. I think it's clearer than the previous one.

I think we should try to use:

* "Email notifications" when it's _the feature_ in particular and when we want to have a short and sweet name
* "Build notifications [via webhooks/email]" when we are talking about the value of having instant build feedback more broadly. It's also really good to say "build notifications via webhook" rather than just "webhooks" since the latter can already mean 3 different types of webhooks... incoming (our API), outgoing (GitHub's API) and build notifications (calling other webhooks with payload templates).

Oh yes, and of course we _also_ say "build status webhooks" :upside_down_face: Not on the website, it just says "webhooks" :)

I think we should remove "build status webhooks" and say "build notifications via webhooks" in general. It's not any big disaster, though, anything containing the word "webhook" is bound to be a bit technical :)